### PR TITLE
fix(mousetrap): correct path for extensions

### LINKED ISSUE #52437: [@types/mousetrap] Property 'bindGlobal' does not exist on type 'MousetrapStatic' after update to 1.6.7
After upgrading `@types/mousetrap` from 1.6.6 to 1.6.7 the types for the global binding are no longer available.

- [x] I tried using the `@types/mousetrap` package and had problems.
- [x] I tried using the latest stable version of tsc. https://www.npmjs.com/package/typescript
- [x] I understand that `@types` package versions [do not follow SemVer](https://github.com/DefinitelyTyped/DefinitelyTyped#how-do-definitely-typed-package-versions-relate-to-versions-of-the-corresponding-library).
- [x] I have a question that is inappropriate for [StackOverflow](https://stackoverflow.com/).  (Please ask any appropriate questions there).
- [x] [Mention](https://github.com/blog/821-mention-somebody-they-re-notified) the authors (see `Definitions by:` in `index.d.ts`) so they can respond.
  - Authors: @peterblazejewicz

### PR DESCRIPTION
**Pull request by peterblazejewicz**
The path should read 'plugins', not 'extensions'.

/cc @doberkofler

Fixes #52437

- [x] Use a meaningful title for the pull request. Include the name of the package modified.
- [x] Test the change in your own code. (Compile and run.)
- [x] [Add or edit tests](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#my-package-teststs) to reflect the change.
- [x] [Run `npm test <package to test>`](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#running-tests).

### CODE REVIEW

File: types/mousetrap/plugins/pause/mousetrap-pause.d.ts [hunk 1]
```diff
@@ -1,4 +1,4 @@
-import { MousetrapStatic, MousetrapInstance } from "../";
+import { MousetrapStatic, MousetrapInstance } from "../..";
```
Comment (alanhchoi) [inline types/mousetrap/plugins/pause/mousetrap-pause.d.ts:1]:
```suggestion
import { MousetrapStatic, MousetrapInstance } from "../../";
```
I think it'd be better to keep the consistency.

**Review by alanhchoi — APPROVED** [review]
LGTM

**Review by alanhchoi — APPROVED** [review]
LGTM!

**Review by orta — APPROVED** [review]

**Comment by peterblazejewicz** [comment]
Ready to merge 💘

**Pull request merged**